products/sle/main.pm

```diff
@@ -810,11 +810,13 @@ elsif (get_var("VIRT_AUTOTEST")) {
             loadtest "virt_autotest/virsh_external_snapshot";
         }
     }
-    elsif (get_var("ENABLE_SRIOV_NETWORK_CARD_PCI_PASSSHTROUGH")) {
-        loadtest "virt_autotest/restore_guests";
+    #those tests which test extended features, such as hotpluggin, virtual network and SRIOV passhthrough etc.
+    #they can be seperated from prj1 if needed
+    elsif (get_var("CHAINED_VIRT_FEATURE_TEST")) {
```

Comment:
Better. openqa has start_after and start_directly_after two kinds of chains, is it possible to use DIRECT_CHAINED_VIRT_FEATURE_TEST

Reply:
Cool